.github/workflows/backport.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
     steps:
       - name: Generate GitHub Token
         id: generate_token
-        uses: tibdex/github-app-token@v1
+        uses: tibdex/github-app-token@b62528385c34dbc9f38e5f4225ac829252d1ea92 # v1.8.0
```

Comment:
Can we refer to the action by tag instead of SHA?

Reply:
GitHub tags are mutable -- so that wouldn't improve our security posture at all. Unfortunately we don't have the nice go.mod/go.sum division for GHA.

However, dependabot is smart enough to keep the tag comments up to date:

https://github.com/dependabot/dependabot-core/pull/5951

So we do know which tag the hash points to, and the two will stay in sync (as long as it is managed by automation).